lib/S3Plugin.js

```diff
@@ -162,20 +162,21 @@ class S3Plugin {
 
     const filename = this._stringArg(request, 'filename'),
       uploadDir = this._stringArg(request, 'uploadDir'),
+      Bucket = this._stringArg(request, 'bucketName', this.config.bucketName),
```

Comment:
Variables should not start with a capital letter

Reply:
I've done changes and I've merged this PR onto the otherer see closing comment